sites/public/src/pages/api/adapter/[...backendUrl].ts

```diff
@@ -53,8 +56,11 @@ export default async (req: NextApiRequest, res: NextApiResponse) => {
     res.setHeader("Set-Cookie", cookies)
     res.statusMessage = response.statusText
     res.status(response.status).json(response.data)
+    if (response.status >= 400) {
+      logger.error(`${req.method} - ${backendUrl} - ${response.status} - ${response.statusText}`)
+    }
   } catch (e) {
-    console.error(
+    logger.error(
```

Comment:
Can we call the logger and the console here?

Reply:
the logger is currently configured to be a console logger already (although it can be redirected to a bunch of options). The main benefit here is 1) more detailed logging 2) better formatting for cloudwatch and, eventually 3) support for opentelemetry